ndb/src/google/cloud/ndb/model.py

```diff
@@ -2241,9 +2252,240 @@ def __init__(self, *args, **kwargs):
 
 
 class KeyProperty(Property):
-    __slots__ = ()
+    """A property that contains :class:`.Key` values.
 
-    def __init__(self, *args, **kwargs):
+    The constructor for :class:`KeyProperty` allows at most two positional
+    arguments. Any usage of :data:`None` as a positional argument will
```

Comment:
The `None` filtering is weird. Any idea what that's about?

Reply:
Using

```python
KeyProperty(None)
```

was billed as a way to explicitly set `name=None` in the original docstring. But that is already the default, so I'm not sure why it matters.

Reply:
Probably a backwards compatibility thing, then.